### ISSUE TITLE
Handle erroneous residues in parser and visualisation

### ISSUE THREAD

**Issue by touwwouter** [cmbi/pdb-vis#15]
The WHAT IF list files may contain residues for which certain properties could not be calculated, e.g. it makes no sense to calculate accessibility if the residues have atoms with zero occupancy. WHAT IF now includes these residues in the acc list file with the reason ~~between quotes~~. We have to find a way to show this to the user.

Suggestion:
- The 'reason' not in the parsed accessibility value range, and should therefore not be within the color range. Instead, the residue should have a distinct color saying "I don't have an acc value".
- A tooltip can say WHY (the reason) doesn't have an acc value.
- The acc values passed to cmbi/proseqjs must either be the acc values or the reason
